### DISCUSSION TITLE
220 def _raise_param_error(param_name: str, param_value: Any) -> None: --> 221     raise ParamError(message=f"`{param_name}` value {param_value} is illegal")

### DISCUSSION THREAD

**Discussion by haozhuang0000** [milvus-io/milvus#34239] [Q&A and General discussion]
Any idea why this is happening while using client.search(...)?

**Comment by haozhuang0000** [comment]
My mistake, got the solution

**Reply by Spiz404** [reply]
I'm facing the same issue, how do you solved it?

**Reply by haozhuang0000** [reply]
@Spiz404 Hi, could you share me the code? It was a while ago, I don't remember how I solve this.
